paper_io_sim.tex

```diff
@@ -723,7 +727,7 @@
             reads can benefit from cache hits.
 
             Our fourth and final experiment (\textit{Exp Nighres})
-            simulates the Nighres skull stripping workflow
+            simulates the Nighres CRUISE cortical reconstruction workflow
```

Comment:
isn't it just part of it? how about tissue classification? If you're only running parts of this workflow you should say it clearly.

Reply:
As i understand, skull stripping is the first step in tissue classification. 
After that we have cortical depth estimation including 4 steps, we stop at the "CRUISE cortical reconstruction" step.

Reply:
ok, could you link the workflow description in the nighres doc?

Reply:
yes of course